base/applications/cmdutils/find/lang/ro-RO.rc

```diff
@@ -16,8 +16,7 @@ FIND [/V] [/C] [/N] [/I] [/OFF[LINE]] ""șir"" [fișier...]\n\
   /C         Numără liniile ce conțin șirul.\n\
   /N         Numără rândurile afișate, începând cu 1.\n\
   /I         Ignoră diferențele între majuscule și minuscule.\n\
-  /OFF[LINE] Nu omite fișierele care care au configurate atributul\n\
-             offline (deconectat)."
+  /OFF[LINE] Nu omite fișierele care au setate atributul offline (deconectat)."
```

Comment:
Could you please show a screenshot of how the `FIND` command line utility looks before and after this change? Personally I'm not that keen of this change so I want to see the actual benefits at hand.

Reply:
I can try to do a screenshot if you want. The benefit is that I fixed a typo. The word "care" was repeated two times. By removing it once the number of characters was reduced.

Reply:
Before

![find command before (test)](https://github.com/reactos/reactos/assets/113553683/63d9cb26-4851-4baa-bd0f-9b971c7cff35)


After

![find command after (test)](https://github.com/reactos/reactos/assets/113553683/a653325c-3282-4fe6-8fc9-7445d941a56f)

Reply:
I think I understand what you mean. I will let the description on two rows as before.

Reply:
>The benefit is that I fixed a typo. The word "care" was repeated two times. By removing it once the number of characters was reduced.

Ah sorry I haven't seen this. Yeah this is good. 👍

Reply:
So, should be the text again on two rows or only on one?

Reply:
Two rows.

Reply:
I see, thank you!